Common/Tests/Utilities.UI/UI/SolutionExplorerTree.cs

```diff
@@ -194,11 +194,23 @@ private AutomationElement FindChildOfProjectHelper(EnvDTE.Project project, strin
             }
             var projElement = projElements.Cast<AutomationElement>().Single();
 
-            var itemElement = path.Any() ? FindNode(
-                projElement.FindAll(TreeScope.Children, Condition.TrueCondition),
-                path,
-                0
-            ) : projElement;
+            AutomationElement itemElement = null;
+            int maxAttempts = 10;
+            int delayMs = 500;
+            for (int attempt = 0; attempt < maxAttempts; attempt++)
```

Comment:
adding retires to improve flakyness

Reply:
Maybe the retry should be in the FindNode?

Reply:
ya good point i'll try to move it

Reply:
updated. added retry to "expandNode" call inside findNode